new-frontiers-14/frontier-station-14
# Meteor Defense Turret

## About the PR
- Added faction and `MobState` to meteors.
- Added meteor defense turret.
> A Point Defence Automated Laser Turret (PDALT). A laser turret designed for protection of vessels from space debris. Use screwdriver to disassemble.

To do:
- [ ] Figure out what combination of turret rotation speed, projectile speed and meteor cloud density wouldn't result in shuttle annihilations.
- [ ] Make a special projectile for the turrets with lower damage.

## Why / Balance
Space hazards are fun

## How to test
1. Spawn, buy a shuttle, install couple PDALT on the outside.
2. Plan a route.
3. Litter planned route with meteors and space dust.
4. Start flying on the planned route.
5. Observe the fireworks.

## Media


## Breaking changes
Shuttle hull

**Changelog**
:cl: 
- add: Added meteor defense turret.
